### ISSUE TITLE
Trying to subset the 'plus' glyph U+002B in Iosevka ≥ 2.1.0 throws an error in pyftsubset

### ISSUE THREAD

**Issue by kramred** [be5invis/Iosevka#348]
Hello,

I ran into a weird issue trying to subset glyphs in [Iosevka](https://github.com/be5invis/Iosevka): whenever the `+`  'plus' glyph `U+002B` was included [`pyftsubset`](https://github.com/fonttools/fonttools) threw an error. I tried a few versions of the font and it seems to appear in version 2.1.0 (and above) – I used the releases from github and did not try to build the font myself.

Command:
```bash
pyftsubset --verbose v2.1.0/iosevka-ss04-regular.ttf --unicodes="U+002B"
```
This is the output for [Iosevka v2.1.0](https://github.com/be5invis/Iosevka/releases/tag/v2.1.0):
```
Text: ''
Unicodes: [43]
Glyphs: []
Gids: []
maxp pruned
cmap pruned
post pruned
GPOS pruned
GSUB pruned
name pruned
glyf pruned
Added gid0 to subset
Closing glyph list over 'GSUB': 2 glyphs before
Glyph names: ['.notdef', 'plus']
Glyph IDs:   [0, 14]
Traceback (most recent call last):
  File "/usr/local/bin/pyftsubset", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/dist-packages/fontTools/misc/loggingTools.py", line 375, in wrapper
    return func(*args, **kwds)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 2879, in main
    subsetter.subset(font)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 2681, in subset
    self._closure_glyphs(font)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 2535, in _closure_glyphs
    font['GSUB'].closure_glyphs(self)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 1422, in closure_glyphs
    self.table.LookupList.Lookup[i].closure_glyphs(s)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 1179, in closure_glyphs
    st.closure_glyphs(s, cur_glyphs)
  File "/usr/local/lib/python3.7/dist-packages/fontTools/subset/__init__.py", line 986, in closure_glyphs
    pos_glyphs = frozenset(r.InputCoverage[seqi].intersect_glyphs(s.glyphs))
  File "/usr/local/lib/python3.7/dist-packages/fontTools/ttLib/tables/otBase.py", line 547, in __getattr__
    raise AttributeError(attr)
AttributeError: InputCoverage
```

And for [Iosevka v2.0.2](https://github.com/be5invis/Iosevka/releases/tag/v2.0.2):

```bash
pyftsubset --verbose v2.0.2/iosevka-ss04-regular.ttf --unicodes="U+002B"
```

```
Text: ''
Unicodes: [43]
Glyphs: []
Gids: []
maxp pruned
cmap pruned
post pruned
GPOS pruned
GSUB pruned
name pruned
glyf pruned
Added gid0 to subset
Closing glyph list over 'GSUB': 2 glyphs before
Glyph names: ['.notdef', 'plus']
Glyph IDs:   [0, 14]
Closed glyph list over 'GSUB': 2 glyphs after
Glyph names: ['.notdef', 'plus']
Glyph IDs:   [0, 14]
Closing glyph list over 'glyf': 2 glyphs before
Glyph names: ['.notdef', 'plus']
Glyph IDs:   [0, 14]
Closed glyph list over 'glyf': 2 glyphs after
Glyph names: ['.notdef', 'plus']
Glyph IDs:   [0, 14]
Retaining 2 glyphs
head subsetting not needed
hhea subsetting not needed
maxp subsetting not needed
OS/2 subsetting not needed
hmtx subsetted
cmap subsetted
fpgm subsetting not needed
prep subsetting not needed
cvt  subsetting not needed
loca subsetting not needed
post subsetted
gasp subsetting not needed
GDEF subsetted
GPOS subsetted
GSUB subsetted
name subsetting not needed
glyf subsetted
head pruned
OS/2 Unicode ranges pruned: [0]
glyf pruned
GDEF pruned
GPOS pruned
GSUB pruned
Input font: 388344 bytes: v2.0.2/iosevka-ss04-regular.ttf
Subset font:   5316 bytes: v2.0.2/iosevka-ss04-regular.subset.ttf
```

I also ran `ttx` on both versions and compared the output (searched for 'plus' and some manual inspection) – but I'm no expert on fonts and could not make out a possible cause for this.

Not sure if this is an issue of [`pyftsubset`](https://github.com/fonttools/fonttools) or the Iosevka font, version ≥ 2.1.0 – I'll post this in both repos.

Any hint on how to fix this would be appreciated. I'll use v2.0.2 to subset the glyphs for my web font for now.

Also: Thanks for making this amazing font! - I'm a big fan.

**Comment by kramred** [comment]
The new Iosevka version 2.3.0 fixes this issue!
🎉

**Comment by kramred** [comment]
Issue fixed by Iosevka version 2.3.0
